app/javascript/packages/form-steps/form-steps.tsx

```diff
@@ -242,10 +237,8 @@ function FormSteps({
   }
 
   const unknownFieldErrors = activeErrors.filter((error) => !fields.current[error.field]?.element);
-  const isValidStep = step.validator?.(values) ?? true;
   const hasUnresolvedFieldErrors =
     activeErrors.length && activeErrors.length > unknownFieldErrors.length;
-  const canContinueToNextStep = isValidStep && !hasUnresolvedFieldErrors;
```

Comment:
This could prove to be a useful detail for future work, but... we don't need it now 🤷  Can always bring it back later.